app/services/nomis/api.rb

```diff
@@ -24,6 +24,19 @@ def initialize
       end
     end
 
+    def lookup_offender_location(offender_id:)
+      response = @pool.with do |client|
+        client.get("/offenders/#{offender_id}/location")
+      end
+    rescue Nomis::APIError => e
+      Raven.capture_exception(e, fingerprint: %w[nomis api_error])
+      PVB::Instrumentation.append_to_log(lookup_offender_location: false)
+      Nomis::Establishment.new
+    else
+      PVB::Instrumentation.append_to_log(lookup_offender_location: true)
+      Nomis::Establishment.new(response.fetch('establishment') { {} })
```

Comment:
How come the fetch is needed? Does the api not return a response with an `establishment` key when is successful?